Fix "kubeconfig_file does not exist"

superbrothers opened the pull request:
This PR fixes the bug that "kubeconfig_file does not exist". If using file path as param in the out script, it must change the working directory to the directory containing the build's full set of sources as the first argument.

/ref #16, #18

inajob commented:
The line below change directory again. I'm afraid this is bug.

https://github.com/zlabjp/kubernetes-resource/blob/327aeccc840090d91e4984aee32020c3bdef4dff/assets/out#L44

superbrothers commented:
@inajob I completely forgot that I had already changed the current directory to the source directory. :sweat:
I fixed to change the current directory before running `kubectl_setup` at 1a7dbe1. Please review the change. Thanks.
